import-export-cli/cmd/root.go

```diff
@@ -159,6 +160,23 @@ func createConfigFiles() {
 		if err != nil {
 			utils.HandleErrorAndExit("Error creating default api spec file", err)
 		}
+	} else {
+		data, err := ioutil.ReadFile(utils.DefaultAPISpecFilePath)
+		if err != nil {
+			utils.HandleErrorAndExit("Error creating default api spec file", err)
+		}
+		defaultApiFile := make(map[string]interface{})
+		if err := yaml.Unmarshal(data, &defaultApiFile); err != nil {
+			utils.HandleErrorAndExit("Error reading controller-config", err)
```

Comment:
defalt api yaml?

Reply:
The file we read here is `default_api.yaml`